### ISSUE TITLE
Command Answers in multiple messages

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#93]
```
I thought it would be nice to have every "message" send as a actual message :) 
This works better with clients which format messages in a 
iphone-bubbles-like-way.

Have a look:
http://pastebin.com/BkvSFFE6
```

Original issue reported on code.google.com by `ThDietri...@gmail.com` on 10 Feb 2011 at 2:03

**Comment by GoogleCodeExporter** [comment]
```
I've split the messages in purpose. When we search sms from all users, that 
could take a long time.
So I send a message for each contact.

This option will be added after a setting refactoring (too much option for now, 
we have to add categories).
```

Original comment by `Florent....@gmail.com` on 10 Feb 2011 at 3:06
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Yes, first we have to redesign the settings, as it's getting pretty confusing.
Isn't you patch missing a if(!seperateMessages) statement for the last send()?
```

Original comment by `fschm...@gmail.com` on 10 Feb 2011 at 6:22

**Comment by GoogleCodeExporter** [comment]
```
yes exactly. I'm unsure why but I had it exactly like that. Thats also why I 
use a temporary variable for the process ;)

*corrected*
```

Original comment by `ThDietri...@gmail.com` on 10 Feb 2011 at 7:09

**Comment by GoogleCodeExporter** [comment]
```
btw. I'm not a 100% sure if the clear() method is sufficient or if a reset of 
other members (Fonts etc) is needed. 
```

Original comment by `ThDietri...@gmail.com` on 10 Feb 2011 at 7:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 11 Feb 2011 at 8:27

**Comment by GoogleCodeExporter** [comment]
```
I have made the change, but still the setting is missing. But now you can build 
your own gtalksms on base of the latest changes. See: 
http://code.google.com/p/gtalksms/source/detail?r=f205c43f37b4fe1335161556701b55
442ff89ddd
```

Original comment by `fschm...@gmail.com` on 11 Mar 2011 at 3:01
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Preferences -> Application settings -> Advanced -> Separate Messages
```

Original comment by `fschm...@gmail.com` on 11 Mar 2011 at 4:26
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v2.1
```

Original comment by `Florent....@gmail.com` on 14 Mar 2011 at 10:42
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
